### PR TITLE
Fix part of #210 : Add Tests for NumericInputIsLessThanRuleClassifierProvider

### PR DESCRIPTION
**Pull request by prayutsu**
## Explanation

Fixes part of #210 
Added tests for ```NumericInputIsLessThanRuleClassifierProvider.kt```

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by prayutsu** [comment]
@rt4914 PTAL

**Comment by rt4914** [comment]
Adding @anandwana001 and @aggarwalpulkit596 as I won't be able to review this today.

File: domain/src/test/java/org/oppia/domain/classify/rules/NumericInputIsLessThanRuleClassifierProviderTest.kt [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+        .matches(answer = POSITIVE_REAL_VALUE_1_5, inputs = inputs)
+    }
+
+    assertThat(exception)
+      .hasMessageThat()
+      .contains("Expected classifier inputs to contain parameter with name 'x'")
+  }
+
+  @Test
+  fun testRealAnswer_stringInput_throwsException() {
```
Comment (aggarwalpulkit596) [inline domain/src/test/java/org/oppia/domain/classify/rules/NumericInputIsLessThanRuleClassifierProviderTest.kt:176]: Can you add a test with int value as well

Comment (prayutsu) [reply domain/src/test/java/org/oppia/domain/classify/rules/NumericInputIsLessThanRuleClassifierProviderTest.kt:176]: Okay

**Comment by prayutsu** [comment]
@aggarwalpulkit596 PTAL

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @prayutsu

**Comment by rt4914** [comment]
@aggarwalpulkit596 merge this once your comments are resolved. Thanks.

File: domain/src/test/java/org/oppia/domain/classify/rules/NumericInputIsLessThanRuleClassifierProviderTest.kt (hunk 1 above)
Comment (aggarwalpulkit596) [reply domain/src/test/java/org/oppia/domain/classify/rules/NumericInputIsLessThanRuleClassifierProviderTest.kt:176]: Thanks

**Review by aggarwalpulkit596 — APPROVED** [review]

**Pull request merged**